code/modules/mob/living/simple_animal/hostile/bear.dm

```diff
@@ -76,6 +76,30 @@
 	melee_damage_lower=10
 	melee_damage_upper=40
 
+/mob/living/simple_animal/hostile/bear/spare
+	name = "spare bear"
+	desc = "This bear has adapted a form of camouflage from generations of natural selection in which the omnivores scavenge from space stations and their dumpsters. Its golden skin fools card scanners into opening the door."
+	health = 300
+	maxHealth = 300
+	melee_damage_lower = 15
+	melee_damage_upper = 35
+	icon_state = "sparebear"
+	icon_dead = "sparebear_dead"
+	default_icon_floor = "sparebear"
+	default_icon_space = "sparebear"
+
+/mob/living/simple_animal/hostile/bear/spare/getarmor(var/def_zone, var/type)
+	if(type == "laser")
+		return 80
+	return 10
+
+/mob/living/simple_animal/hostile/bear/spare/getarmorabsorb()
+	return 25
+
+/mob/living/simple_animal/hostile/bear/spare/GetAccess()
+	var/datum/job/captain/J = new/datum/job/captain
```

Comment:
Either make this a var on the bear itself or make it static so that you aren't creating a new job datum *every time* this is called

Reply:
Resolved. Went straight for the source

Reply:
Even better. Thanks